entrypoint.sh

```diff
@@ -19,6 +19,10 @@ args=$(echo "$@" | xargs)
 GIT_CLIFF_OUTPUT="$OUTPUT" git-cliff $args
 exit_code=$?
 
+# Retrieve context
+CONTEXT="$(mktemp)"
+GIT_CLIFF_OUTPUT="$CONTEXT" git-cliff $args --context
```

Comment:
Here, we're running `git-cliff` a second time to retrieve the context. I haven't dug much into this, but it might be a better idea if there was a way to output the changelog AND the context together.

Reply:
I think this is not possible right now. Feel free to create an issue on the main repo for consideration 🐻